### DISCUSSION TITLE
[Question][Hw5] Is it required to compute N VSAs and intersect them?

### DISCUSSION THREAD

**Discussion by retroinspect** [prosyslab-classroom/cs424-program-reasoning#345] [Q&A]
Is it required to compute N VSAs(each represents all programs satisfying counterpart example) and intersect them if the number of examples provided was N?

Or, is it okay if our **final VSA represents all programs satisfying all examples**, and our **VSA structure concisely represents the set of programs**?

**Comment by KihongHeo** [comment]
The evaluation criteria are the same. We will only consider the final outcome and will not consider the detailed algorithm.

I just introduced the standard method for guidance in the lecture. If you have your own good method, it will be also OK.
